doc/design.txt

```diff
@@ -255,18 +261,15 @@ confirmed as good by the user (0x01=confirmed; 0xff=not confirmed).
 
 *** IMAGE TRAILERS
 
-At startup, the boot loader determines which of the above four states the
-device is in by inspecting the image trailers.  When using the term "image
-trailers" what is meant is the aggregate information provided by both image
-slot's trailers.
+At startup, the boot loader determines the boot swap type by inspecting the
+image trailers.  When using the term "image trailers" what is meant is the
```

Comment:
I think this sentence really just indicates that the following description was awkward. If we take it out, do things still make sense? If not, I think we should fix the descriptions below, rather than try to define what plural means.